sys-auth/elogind/files/elogind-255.5-musl.patch

```diff
@@ -0,0 +1,196 @@
+diff --git a/src/basic/cgroup-util.c b/src/basic/cgroup-util.c
```

Comment:
Rebasing the old patches in `files/` and keeping them split up per reason with annotations/headers like they used to have would be good.

e.g. https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-auth/elogind/files/elogind-252.9-musl-more-strerror_r.patch?id=c87cd090b43ed659314d627f468ac4e341de5197 had some explanation and now it's lost.